spec.html

```diff
@@ -22053,11 +22053,13 @@
               1. Perform ? ModuleDeclarationEnvironmentSetup(_module_).
               1. Assert: _module_ occurs exactly once in _stack_.
               1. Assert: _module_.[[DFSAncestorIndex]] is less than or equal to _module_.[[DFSIndex]].
-              1. If _module_.[[DFSAncestorIndex]] equals _module_.[[DFSIndex]], repeat:
-                1. Let _requiredModule_ be the last element in _stack_.
-                1. Remove this element from _stack_.
-                1. Set _requiredModule_.[[Status]] to `"instantiated"`.
-                1. If _requiredModule_ is _module_, end the repetition.
+              1. If _module_.[[DFSAncestorIndex]] equals _module_.[[DFSIndex]], then
```

Comment:
I personally found the repeat notation fine. Do you prefer the explicit while loop, or is this about consistency with existing forms?

Reply:
Just for consistency with the existing forms.